src/Common/Config/ConfigProcessor.h

```diff
@@ -114,6 +116,9 @@ class ConfigProcessor
     static bool isPreprocessedFile(const std::string & config_path);
```

Comment:
Kind of related: In l. 106 (function `savePreprocessedConfig`), we could pass `loaded_config` by const reference instead of by reference.